src/components/tableWithTwoRecipes.js

```diff
@@ -0,0 +1,107 @@
+// Component image start
+// Config file
+import { config } from "../config.js";
+
+const getImage = (id, href, src) => {
+  return `<a href="${href}" data-link-id="${id}" target="_self">
+<img src="${
+    config.pathImages + src
```

Comment:
you can create a variable before return statement and make this code even less complex